spec/features/h2_object_creation_spec.rb

```diff
@@ -113,12 +113,21 @@
 
     expect(page).to have_text 'You have successfully deposited your work'
 
-    # Opens Argo and searches on title
-    visit Settings.argo_url
-    find_field('Search...').send_keys("\"#{item_title}\"", :enter)
-    reload_page_until_timeout!(text: 'v2 Accessioned')
+    # Opens Argo detail page
+    visit "#{Settings.argo_url}/view/#{bare_druid}"
+    Timeout.timeout(Settings.timeouts.workflow) do
+      loop do
+        break if page.has_text?('v2 Accessioned', wait: 1)
+
+        reset_errored_workflow_step('accessionWF') if page.has_text?('Error: shelve : problem with shelve', wait: 0)
+        sleep 3 # wait to ensure the workflow step reset was accepted so we don't reset again immediately
+        page.refresh
+      end
+    end
 
```

Comment:
this borrows but modifies some logic from the page helper method `reload_page_until_timeout!` to look for the shelve step in error, and then calls out to a new method to reset the error